app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotVaccinatedError, OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+    try:
+        for friend in friends:
+            cafe.visit_cafe(friend)
```

Comment:
you should place  `try` block within the loop, each friend is processed individually, and any exceptions that occur during their visit are isolated